cli/README.md

```diff
@@ -31,4 +31,10 @@ Some examples:
 - Bundle using a custom template and add custom `templateOptions`:<br/>
   `$ redoc-cli build [spec] -t custom.hbs --templateOptions.metaDescription "Page meta description"`
 
+#### With `.redocly.yaml` file ([more info](https://redocly.com/docs/cli/configuration/#redocly-configuration-file)):
+
+1. Go to folder with `.redocly.yaml` and your definition file
```

Comment:
Does it work with `redocly.yaml` file too?

Reply:
Yes, it works too

Reply:
```suggestion
1. Go to folder with your Redocly configuration file (`.redocly.yaml` or `redocly.yaml`) and your OpenAPI definition file.
```